manager/manager/utils.py

```diff
@@ -720,3 +781,243 @@ def assert_time_data(time_data):
+        Signed, your friendly neighborhood observers,
+        {report["observers_crew"]}
+        """
+        return plain_content
+
+    new_line_character = "\n"
+    html_content = f"""
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <style>
+            table {{
+                font-family: Arial, sans-serif;
+                border-collapse: collapse;
+                width: 100%;
+            }}
+            th {{
+                background-color: #f2f2f2;
+            }}
+            th, td {{
+                border: 1px solid #dddddd;
+                text-align: left;
+                padding: 8px;
+            }}
+        </style>
+    </head>
+    <body>
+        <p>
+            Hello everyone!
+            <br>
+            Please find below a summary of the observing night and links for more detailed information.
+        </p>
+        <p>
+            Summary:
+            <br>
+            {report["summary"].replace(new_line_character, '<br>')}
+        </p>
+        <p>
+            Final telescope status:
+            <br>
+            {report["telescope_status"].replace(new_line_character, '<br>')}
+        </p>
+        <p>
+            Additional resources:
+            <br>
+            <ul>
+                <li>
+                    OBS fault reports from last 24 hours:
+                    <a href="{url_jira_obs_tickets}">{url_jira_obs_tickets}</a>
+                </li>
+                <li>
+                    Link to {report["telescope"]} Log Confluence Page:
+                    <a href="{report["confluence_url"]}">{report["confluence_url"]}</a>
+                </li>
+                <li>
+                    Link to detailed night log entries (requires Summit VPN):
+                    <a href="{url_rolex}">{url_rolex}</a>
+                </li>
+            </ul>
+        </p>
+        {f'''<p>
+            Detailed issue report:
+            <br>
+            {parse_obs_issues_array_to_html_table(report["obs_issues"])}
+            <br>
+            Total obstime loss: {sum([issue['time_lost'] for issue in report["obs_issues"]])} hours
+        </p>''' if len(report["obs_issues"]) > 0 else ""}
+        <p>
+            Signed, your friendly neighborhood observers,
+            <br>
+            {", ".join(report["observers_crew"])}
+        </p>
+    </body>
+    </html>
+    """
+
+    return html_content
+
+
+def parse_obs_issues_array_to_html_table(obs_issues):
+    """Parse the OBS issues array to an HTML table.
+
+    Parameters
+    ----------
+    obs_issues : `list`
+        List of OBS issues
+
+    Notes
+    -----
+    Each element of the obs_issues list must be dictionary
+    with the following keys:
+    - key: The key of the issue
+    - summary: The summary of the issue
+    - time_lost: The time lost in hours
+    - reporter: The reporter of the issue
+    - created: The creation date of the issue
```

Comment:
Very nice content for the email. However, there are a lot of strings that are repeated for the plain text and HTML templates. Please extract those in constants and use those constants instead. That way, if at some moment the contents of the emails need to be updated, it only needs to be done in one place. You probably want to open a new ticket for that and add a TODO here with a reference to the ticket.